tools/db_bench_tool.cc

```diff
@@ -2044,6 +2044,7 @@ class Benchmark {
 
   class ErrorHandlerListener : public EventListener {
    public:
```

Comment:
incorrect indentation?

Reply:
I didn't change indentation. Do you mean there shoudn't be spaces before 'class'?

Reply:
yeah it's the old code that had indentation problem, I think it should either indent 0 or 2. right now it's indent 1 which is odd

Reply:
This is following Google C++ Style: https://google.github.io/styleguide/cppguide.html#Class_Format